src/flamegraph.cpp

```diff
@@ -365,7 +365,16 @@ QBrush brushSystem(const Data::Symbol& symbol)
     return user;
 }
 
-QBrush brush(const Data::Symbol& entry, Settings::ColorScheme scheme)
+QBrush improvedBrush(quint32 cost, quint32 totalCost)
```

Comment:
"improved" is very subjective. I don't think it improves anything. please give it a different name, something like `brushByCostRatio` or similar